pmpro-membership-card.php

```diff
@@ -71,6 +74,13 @@ function pmpro_membership_card_wp()
 			wp_redirect(pmpro_url("levels"));
 			exit;
 		}
+		// check if membership cards are permitted for their membership level.
+		$pmpro_level = pmpro_getMembershipLevelForUser();
+		$pmpro_show_membership_card = get_option( 'pmpro_show_membership_card_for_level_' . $pmpro_level->id, 1 );
+		if ($pmpro_show_membership_card !== 1) {
+			wp_redirect( pmpro_url( 'account' ) );

```

Comment:
This may be good to have a filter here, will allow developers to adjust the redirect URL for non card holding levels.

Reply:
Thanks, Andrew. Done in https://github.com/strangerstudios/pmpro-membership-card/pull/35/commits/aefd93a9e9f37823578dedf9abe478b23dfad43b.

Version bump was reversed in https://github.com/strangerstudios/pmpro-membership-card/pull/35/commits/b1754824f8dbb20b5c359f74b0c7c7e16c8495e7.

And i18n was addressed in https://github.com/strangerstudios/pmpro-membership-card/pull/35/commits/789711f6ace1ec60a5070c2dedbc45e10f9551a2.